### PR TITLE
ztp: CNF-14458: Follow up fixes for channel name & kubecompare-references

### PR DESCRIPTION
**Pull request by fontivan**
- Fix channel name to match what should be the final version in the catalog
- Update kube-compare references based on test result feedback

### CODE REVIEW

File: ztp/kube-compare-reference/required/cluster-logging/ClusterLogForwarder.yaml [hunk 1]
```diff
@@ -1,37 +1,26 @@
...
   namespace: openshift-logging
 spec:
   {{- if .spec.outputs }}
   outputs:
-    - type: {{ (index .spec.outputs 0).type }}
-      name: {{ (index .spec.outputs 0).name }}
-      kafka:
-        url: {{ (index .spec.outputs 0).kafka.url }}
+  - name: {{ (index .spec.outputs 0).name }}
+    type: kafka
```
Comment (fontivan) [inline ztp/kube-compare-reference/required/cluster-logging/ClusterLogForwarder.yaml:10]: @imiller0 as far as the kube-compare reference goes should we only allow kafka forwarder or any output?

Comment (imiller0) [reply ztp/kube-compare-reference/required/cluster-logging/ClusterLogForwarder.yaml:10]: For now kafka as that is what we have evaluated the resource utilzation with. Anything outside of that would need additional verification in the RAN DU use case.

**Review by imiller0 — APPROVED** [review]
/lgtm

**Pull request merged**